lib/modelManagers/commentsReactionModel.js

```diff
@@ -0,0 +1,84 @@
+const { ArticlesComments, CommentsReactions } = require('../../database/models');
```

Comment:
Why is `ArticlesComments` been imported here?

Reply:
The article comments model manager has not been merged and I have to check for a valid comment Id before a user can react.

Reply:
I need the model to do that

Reply:
I'll do that